xbmc/interfaces/builtins/GUIBuiltins.cpp

```diff
@@ -564,6 +576,7 @@ CBuiltins::CommandMap CGUIBuiltins::GetOperations() const
            {"setproperty",                    {"Sets a window property for the current focused window/dialog (key,value)", 2, SetProperty}},
            {"setstereomode",                  {"Changes the stereo mode of the GUI. Params can be: toggle, next, previous, select, tomono or any of the supported stereomodes (off, split_vertical, split_horizontal, row_interleaved, hardware_based, anaglyph_cyan_red, anaglyph_green_magenta, anaglyph_yellow_blue, monoscopic)", 1, SetStereoMode}},
            {"takescreenshot",                 {"Takes a Screenshot", 0, Screenshot}},
-           {"toggledirtyregionvisualization", {"Enables/disables dirty-region visualization", 0, ToggleDirty}}
+           {"toggledirtyregionvisualization", {"Enables/disables dirty-region visualization", 0, ToggleDirty}},
+           {"windowshdrswitch",               {"Toggles Windows HDR On/Off and restarts Kodi", 0, WindowsHDRSwitch}}
```

Comment:
Can this also be made more generic and implemented by the different platforms?
Documentation also missing

Reply:
Done in https://github.com/xbmc/xbmc/pull/17012/commits/fae9ab8d6f0f3e5be64b4b108c8eb154f2088785